plugins/web/external/Application_Configuration_Management@OWTF-CM-004.py

```diff
@@ -4,14 +4,9 @@
 https://www.owasp.org/index.php/Testing_for_application_configuration_management_%28OWASP-CM-004%29
 NOTE: GREP plugins do NOT send traffic to the target and only grep the HTTP Transaction Log
 """
-
-import string, re
-import cgi
-
```

Comment:
Blank line(s) b/w docstrings and top-level variable definition (here "DESCRIPTION")